spec/javascripts/services/account_service_spec.coffee

```diff
@@ -68,12 +64,11 @@ do ->
       $provide.value '$translate', $translate
       $provide.value 'bsLoadingOverlayService', fakeLoadingOverlayService
       $provide.value 'ShortFormApplicationService', fakeShortFormApplicationService
-      $provide.value 'ShortFormDataService', fakeShortFormDataService
       $provide.value 'ModalService', fakeModalService
       return
     )
 
-    beforeEach inject((_AccountService_, _$state_, _$auth_, _$q_, _$httpBackend_) ->
+    beforeEach inject((_AccountService_, _ShortFormDataService_, _$state_, _$auth_, _$q_, _$httpBackend_) ->
```

Comment:
pulled in real ShortFormDataService because too much mocking makes me worry about the actual functionality